### ISSUE TITLE
Training DB Text doesn't progress at all 

### ISSUE THREAD

**Issue by bely66** [PaddlePaddle/PaddleOCR#5546]
- 系统环境/System Environment：Linux Ubuntu 18.06
- 版本号/Version：Paddle：2.2.0  PaddleOCR：2.3.0.2 问题相关组件/Related components：
- 运行指令/Command Code：!python3 tools/train.py -c configs/det/det_r50_vd_db_arabic.yml \
                                                     -o Global.pretrain_weights=./pretrain_models/ResNet50_vd_ssld_pretrained

I did the following steps: 
1. Annotated Images for Text Detection
2. Converted the annotations to ICDAR Format (Image_path    [{"transcription":"###", "points": []]), *I filled all the transcription with ###*
3. Generated a config file like train_icdar2015_label.txt to arabic_ocr_train_data.txt
4. Generated a text file like det_r50_vd_db.yml called det_r50_vd_db_arabic.yml 
5. Changed the label_file_list name in det_r50_vd_db_arabic.yml 
6. In Training, Everything is working without error 
7. But the Loss doesn't change as training progress it gives me this output:
 `lr: 0.001000, loss: 1.000000, loss_shrink_maps: 0.000000, loss_threshold_maps: 0.000000, loss_binary_maps: 1.000000`

I'm positive my annotations are right, and tried using the same function to plot ICDAR and my annotations and they match in the format.

Anyone can point me towards what might be wrong?

**Comment by LDOUBLEV** [comment]
Do not fill all transcription with ###. Text boxes with text ### will be ignored during training.

**Comment by bely66** [comment]
Yes, worked

**Comment by youde2000** [comment]
我就说嘛，原来这样
